src/ERC721SeaDrop.sol

```diff
@@ -439,4 +474,103 @@ contract ERC721SeaDrop is
     ) public override onlyAllowedOperator(from) {
         super.safeTransferFrom(from, to, tokenId, data);
     }
+
+    /**
+     * @notice Configure multiple properties at a time.
+     */
+    function multiConfigure(MultiConfigureStruct calldata config)
```

Comment:
Seems like this is only meant for _initially_ configuring parameters of a drop, right? Looks like to un-set/remove any of these values, the normal methods would have to be called. We may want to document that well to avoid confusion/foot-guns, especially things on like `allowedPayers` where old ones won't be un-set.

Reply:
added unsettable params in https://github.com/ProjectOpenSea/seadrop/pull/22/commits/bb59828a64d13cbf0f5f75c3f40ce848a22560e5